components/learn_more_trial_modal/learn_more_trial_modal_step.tsx

```diff
@@ -61,7 +61,7 @@ const LearnMoreTrialModalStep = (
                 <span>
                     <FormattedMarkdownMessage
                         id='start_trial.modal.disclaimer'
-                        defaultMessage='By clicking “Start trial”, I agree to the [Mattermost Software Evaluation Agreement,](!https://mattermost.com/software-evaluation-agreement) [privacy policy,](!https://about.mattermost.com/default-privacy-policy/) and receiving product emails.'
+                        defaultMessage='By clicking “Start trial”, I agree to the [Mattermost Software Evaluation Agreement,](!https://mattermost.com/software-evaluation-agreement) [privacy policy,](!https://mattermost.com/privacy-policy/) and receiving product emails.'
```

Comment:
These strings were added after the previous PR but still pointed to the old address. I was just doing a find-and-replace for all the changes from https://github.com/mattermost/mattermost-webapp/pull/9785, so it caught these too